src/containers/ModalSignIn/Secret/SecretField.tsx

```diff
@@ -64,10 +67,9 @@ export function SecretField(props: SecretFieldProps) {
         name="secret"
         type={showPassword ? 'text' : 'password'}
       />
-      {
-        step === 'password'
-        && <S.ForgottenPasswordLink onClick={resetPassword}>Mot de passe oublié ?</S.ForgottenPasswordLink>
-      }
+      <S.ForgottenPasswordLink onClick={resetPassword}>
+        {step === 'password' ? 'Mot de passe oublié ?' : "Renvoyer le code d'activation"}
```

Comment:
eslint devrait te sortir une erreur sur ça

```suggestion
        {step === 'password' ? 'Mot de passe oublié ?' : 'Renvoyer le code d'activation'}
```

Reply:
C'est eslint qui a mis une double quote car il y avait une apostrophe dans le texte